cmd/testclient/client.go

```diff
@@ -276,6 +279,36 @@ func main() {
 			os.Exit(1)
 		}
 		sugar.Infow("DeleteCluster method", "response", v)
+	case "DeleteAllClustersInRegion":
+		v, err := client.GetClusters(context.Background(), getClustersReq)
+		if err != nil {
+			sugar.Errorw("error getting clusters", "account", getClustersReq.AccountName, "provider", getClustersReq.Provider, "region", getClustersReq.Region, "error", err)
+			os.Exit(1)
+		}
+		clusters := make([]string, 0)
+		for _, cluster := range v.Clusters {
+			clusters = append(clusters, cluster.Name)
+		}
+		logger.Sugar().Infow("deleting following clusters", "clusters", clusters)
+
+		for _, cluster := range clusters {
+			sugar.Infow("deleting cluster", "cluster", cluster)
+			req := &proto.ClusterDeleteRequest{
+				Provider:    provider,
+				Region:      region,
+				AccountName: accountName,
+				ClusterName: cluster,
+				ForceDelete: true,
+			}
+			v, err := client.DeleteCluster(context.Background(), req)
+			if err != nil && err.Error() != "" {
+				sugar.Errorw("error deleting cluster", "cluster", cluster, "error", err)
+				os.Exit(1)
```

Comment:
nitpick: avoid using os.Exit unless its absolutely necessory. in large application, there are paths which rely on defer calls, Exit wont allow that to execute.

Reply:
This will only exit the testclient cli app and not spawner application

Reply:
yes, Its just a suggestion. here return also would work the same.

Reply:
Removed it altogether so we can try and delete other clusters.